p2p/test_util.go

```diff
@@ -319,13 +319,17 @@ func (mockNodeInfo) Handshake(net.Conn, time.Duration) (ni.NodeInfo, error) { re
 
 func testNodeInfo(id nodekey.ID, name string) ni.Default {
```

Comment:
Still `nodekey` here, don't think it changes anything.

Reply:
But this is `p2p` package...